lib/service/config.go

```diff
@@ -11,6 +11,7 @@ type Config struct {
 	LNDMacaroonHex        string `envconfig:"LND_MACAROON_HEX" required:"true"`
 	LNDCertHex            string `envconfig:"LND_CERT_HEX"`
 	CustomName            string `envconfig:"CUSTOM_NAME"`
+	Host                  string `envconfig:"HOST" default:"localhost:3000"`
```

Comment:
Yes, this is only used for swagger. If this is set in dev / prod, for us it will be either `lndhub.regtest.getalby.com` or `ln.getalby.com`, so withouth the port. So it doesn't really make sense to combine this with the port environment variable.